Abstract (#14)
astropy/astropy-v5.0-paper: src/ms.tex

% \input{author.tex}

\begin{abstract}
The \astropy Project supports and fosters the development of open-source and openly-developed
\python packages that provide commonly-needed functionality to the astronomical
community.
A key element of the \astropy Project is the core package \astropypkg, which serves as the
foundation for more specialized projects and packages.
In this article, we summarize key features in the core package as of the recent major
release, version 5.0, and provide major updates for the project.
We then discuss supporting a broader ecosystem of inter-operable packages,

> including connections with several observatories and missions.

Reviewer: ```suggestion
including connections with several astronomical observatories and missions.
```

We also revisit the future outlook of the \astropy Project and the current

> status of Learn Astropy. Last but not least, we contemplate the current
> and future challenges facing the project.

Reviewer: ```suggestion
status of Learn Astropy. 
We conclude by raising and discussing the current and future challenges facing the project.
```
